travis.py

```diff
@@ -36,6 +36,7 @@ def main():
 
     run_build(buildInfo)
 
+
```

Comment:
Can you revert it please?

Reply:
I added it for best practices purpose: Python interpretor needs two blank lines before a new function declaration (same as the blank line at the end of the file).